tapir/coop/views/shareowner.py

```diff
@@ -684,7 +684,9 @@ def __init__(self, *args, **kwargs):
     def shift_slot_filter(queryset: ShareOwner.ShareOwnerQuerySet, name, value: str):
         return queryset.filter(
             # Find all Tapir-Users currently enrolled in that shift-name "value"
-            user__in=TapirUser.objects.registered_to_shift_slot_name(value)
+            user__in=Shift.objects.filter(
+                start_time__gt=timezone.now(), slots__name=value
+            ).values("slots__attendances__user")
```

Comment:
Is this filter doing the correct thing? Would it be possible to write a test for it?

Reply:
Good point. We have one, but it still filters for SlotTemplate-created shifts only

Reply:
Not sure why, but the pipeline didn't start the test, so we don't see it failing.

Reply:
Ah, looks like the tests fail on your repo: https://github.com/crosspolar/tapir/actions/runs/7678998537/job/20929424550. That's a bit strange of GitHub.

Reply:
fixed. 
Failed because the filter redirects to the user page if only one user is in the queryset.